x/incentives/keeper/distribute.go

```diff
@@ -713,6 +698,28 @@ func (k Keeper) distributeInternal(
 	return totalDistrCoins, err
 }
 
+func (k Keeper) skipSpamGaugeDistribute(ctx sdk.Context, locks []*lockuptypes.PeriodLock, gauge types.Gauge, totalDistrCoins sdk.Coins, remainCoins sdk.Coins) (bool, sdk.Coins, error) {
+	if len(locks) == 0 {
+		return true, nil, nil
+	}
+
+	// In this case, remove redundant cases.
+	// Namely: gauge empty OR gauge coins undistributable.
+	if remainCoins.Empty() {
+		ctx.Logger().Debug(fmt.Sprintf("gauge debug, this gauge is empty, why is it being ran %d. Balancer code", gauge.Id))
+		err := k.updateGaugePostDistribute(ctx, gauge, totalDistrCoins)
+		return true, totalDistrCoins, err
+	}
+
+	// Remove some spam gauges that are not worth distributing. (We ignore the denom stake because of tests.)
```

Comment:
> (We ignore the denom stake because of tests.)

LOL, cant we just inc the tests to be greater than 100??

Reply:
yeah but its pain